src/Service.Tests/dab-config.MsSql.json

```diff
@@ -19,4006 +19,2989 @@
-                "exclude": [],
-                "include": ["*"]
-              },
-              "policy": {
-                "request": null,
-                "database": "@item.id eq 1940"
-              }
-            },
-            {
-              "action": "update",
-              "fields": {
-                "exclude": [],
-                "include": ["*"]
-              },
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "create",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "delete",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            }
-          ]
-        },
-        {
-          "role": "policy_tester_06",
-          "actions": [
-            {
-              "action": "read",
-              "fields": {
-                "exclude": [],
-                "include": ["*"]
-              },
-              "policy": {
-                "request": null,
-                "database": "@item.id eq 1940"
-              }
-            },
-            {
-              "action": "update",
-              "fields": {
-                "exclude": [],
-                "include": ["*"]
-              },
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "create",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "delete",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            }
-          ]
-        },
-        {
-          "role": "database_policy_tester",
-          "actions": [
-            {
-              "action": "create",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": "@item.name ne \u0027New publisher\u0027"
-              }
-            },
-            {
-              "action": "update",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": "@item.id ne 1234"
-              }
-            },
-            {
-              "action": "read",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": "@item.id ne 1234 or @item.id gt 1940"
-              }
-            }
-          ]
+  "source": {
+    "object": "publishers",
+    "type": "table"
+  },
+  "graphql": {
+    "enabled": true,
+    "operation": null,
```

Comment:
Is this another property that can be dropped if null?

Reply:
Going back to your previous comment about writing nulls, looks like you have to explicitly skip it if you're doing your own converter. The `EntityGraphQLConverter` is causing that.